config/schema/document_types/asylum_support_decision.json

```diff
@@ -0,0 +1,190 @@
+{
+  "fields": [
+    "tribunal_decision_decision_date",
+    "tribunal_decision_judges",
+    "tribunal_decision_category",
+    "tribunal_decision_sub_category",
+    "tribunal_decision_landmark",
+    "tribunal_decision_reference_number"
+  ],
+  "allowed_values": {
+    "tribunal_decision_judges": [
```

Comment:
Agreed that it's not great. Depending on when this happens I'll be travelling / remote so it would be nice if I could dial in or be kept in the loop about the discussion.